server/controllers/checkAuth.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require('jsonwebtoken');
+const cookie = require('cookie');
+
+const { SECRET } = process.env;
+
+exports.checkAuth = (req, res) => {
+  if (req.headers.cookie) {
+    const token = cookie.parse(req.headers.cookie).logged_in;
+    jwt.verify(token, SECRET, (err) => {
+      if (err) {
+        return res.status(401).json({ error: 'something went wrong!' });
+      }
+      return res.json({ message: 'success' });
```

Comment:
if you make ```success: true``` here, if front-end you can check if there's message or not

Reply:
done :+1:

Reply:
Yeah make sure all of these endpoints are returning the same structure 👍